tests/cases/expr/abs.60

```diff
@@ -0,0 +1,35 @@
+// Copyright © SixtyFPS GmbH <info@sixtyfps.io>
+// SPDX-License-Identifier: (GPL-3.0-only OR LicenseRef-SixtyFPS-commercial)
+
+ TestCase := Rectangle {
+    property<float> t1: abs(42.3);
+    property<float> t2: abs(-42.3);
+    property<int> t3: abs(42.3);
+    property<int> t4: abs(-42.3);
+}
```

Comment:
I'm sure you saw that there was already tests for all these functions in `math.60` and `trigo.60` and the like.
I'm not against having more tests, quite the contrary. But each tests has additional overhead in C++ to rerun the compiler which is slow. Same for node.  So maybe grouping similar tests in the same file can speedup the test executions.

Also, for the interpreter test, it is good to have a `property <bool> test: ....` that evaluates to true. (the interpreter is also tested by the nodejs anyway, but the interpreter tests being much faster, it's good to catch problem in the interpreter test itself)